### ISSUE TITLE
0.17.0 release is not working when sourced from unpkg.com

### ISSUE THREAD

**Issue by domibarton** [excalidraw/excalidraw#7362]
Bug
===

When following [the official docs](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/integration#browser) of using Excalidraw in a browser, I get this error in the JS console:

```
Uncaught TypeError: u is undefined
    8935 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    5992 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    7766 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    4048 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    3528 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    9645 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    7422 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    4761 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    9660 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    6049 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    4949 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    2361 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    4682 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    6264 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    457 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    583 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    7935 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    7651 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    4726 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    4105 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    2448 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    5463 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    8242 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    4864 https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    s https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    <anonymous> https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    <anonymous> https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    <anonymous> https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    <anonymous> https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
    <anonymous> https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js:2
excalidraw.production.min.js:2:737982
```

Using the development release also has some issues:

```
Uncaught TypeError: react_jsx_runtime__WEBPACK_IMPORTED_MODULE_0__ is undefined
    <anonymous> icons.tsx:74
    tsx excalidraw.development.js:3588
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> Hyperlink.tsx:21
    tsx excalidraw.development.js:3852
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> renderScene.ts:22
    ts excalidraw.development.js:4424
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> transformHandles.ts:13
    ts excalidraw.development.js:4050
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> collision.ts:29
    ts excalidraw.development.js:3885
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> binding.ts:19
    ts excalidraw.development.js:3863
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> linearElementEditor.ts:10
    ts excalidraw.development.js:3940
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> textElement.ts:47
    ts excalidraw.development.js:4028
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> renderElement.ts:21
    ts excalidraw.development.js:4413
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> Shape.ts:8
    ts excalidraw.development.js:4490
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> bounds.ts:23
    ts excalidraw.development.js:3874
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> math.ts:27
    ts excalidraw.development.js:4303
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> frame.ts:30
    ts excalidraw.development.js:4094
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> selection.ts:14
    ts excalidraw.development.js:4567
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> Scene.ts:7
    ts excalidraw.development.js:4479
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> mutateElement.ts:7
    ts excalidraw.development.js:3951
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> sizeHelpers.ts:10
    ts excalidraw.development.js:4006
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> index.ts:51
    ts excalidraw.development.js:3929
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> scrollbars.ts:9
    ts excalidraw.development.js:4556
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> index.ts:19
    ts excalidraw.development.js:4534
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> actionDeleteSelected.tsx:7
    tsx excalidraw.development.js:2125
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> index.ts:67
    ts excalidraw.development.js:2367
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> App.tsx:23
    tsx excalidraw.development.js:2510
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> index.tsx:56
    tsx excalidraw.development.js:4336
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> entry.js:54
    js excalidraw.development.js:4325
    __webpack_require__ excalidraw.development.js:5614
    <anonymous> excalidraw.development.js:5878
    <anonymous> excalidraw.development.js:5881
    webpackUniversalModuleDefinition excalidraw.development.js:17
    <anonymous> excalidraw.development.js:18
icons.tsx:74:40
```

Workaround
=========

Please note, I have caching completely disabled and just copy-pasted the snippet. Downgrading & pinning to `0.16.1` fixes the issue:

```html
<script src="https://unpkg.com/@excalidraw/excalidraw@0.16.1/dist/excalidraw.production.min.js" type="text/javascript"></script>
```

Finding
======

Problem is with the `0.17.0` release.

**Comment by ad1992** [comment]
@domibarton a patch was recently released to fix please use `0.17.1`

**Comment by ad1992** [comment]
Closing this, if it persists with `0.17.1`, please reopen the issue.
